command/connect/redirecttraffic/redirect_traffic.go

```diff
@@ -50,6 +51,7 @@ type cmd struct {
 func (c *cmd) init() {
 	c.flags = flag.NewFlagSet("", flag.ContinueOnError)
 
+	c.flags.StringVar(&c.consulDNSIP, "consul-dns-ip", "", "IP used to reach the Consul DNS server. If provided, DNS queries will be redirected to Consul.")
```

Comment:
To avoid confusion with consul server
```suggestion
	c.flags.StringVar(&c.consulDNSIP, "consul-dns-ip", "", "IP used to reach Consul DNS. If provided, DNS queries will be redirected to Consul.")
```